user_docs/en/userGuide.t2t

```diff
@@ -101,8 +101,8 @@ Check the [System Requirements #SystemRequirements] for full details.
 These steps assume some familiarity with navigating a web page.
 
 - Open your web browser (Press the ``Windows`` key, type the word "internet" without quotes, and press ``enter``)
-- Load the NV Access download page (Press ``alt+d``, type the following address and press ``enter``): 
-https://www.nvaccess.org/download 
+- Load the NV Access download page (Press ``alt+d``, type the following address and press ``enter``):

```

Comment:
> Most of the diff to the user guide seems to be just trailing white spaces removal. This makes it hard to spot real changes, and also may cause these lines to show up as requiring re-translations for translators. I believe these changes should be reverted.

I fully agree with it.
@leonardder, if not already done, could you please remove automatic space removing from your work configuration (pre-commit hook?). Since at least two PRs were impacted by this issue, could you also double check the PRs and branches where you have been working recently (after having introduced this config)? Thanks!